src/components/MembersSectionNew/data/data.ts

```diff
@@ -0,0 +1,137 @@
+export const memberData = [
```

Comment:
do we need this file ?

Reply:
it will be used temporary, will change the location of the file